src/main/java/com/jvm_bloggers/core/data_fetching/http/ProtocolSwitchingAwareConnectionRedirectHandler.java

```diff
@@ -80,8 +80,8 @@ public HttpURLConnection handle(@NonNull URLConnection urlConnection,
                     return conn;
             }
 
-        } while (true);
-
+        }
+        while (true);
```

Comment:
Maybe this is not the best possible solution. I'll adjust checkstyle rules according to https://github.com/checkstyle/checkstyle/issues/3320